docs/rules/no-mixed-operators.md

```diff
@@ -5,6 +5,8 @@ This rule warns when different operators are used consecutively without parenthe
 
 ```js
 var foo = a && b || c || d;    /*BAD: Unexpected mix of '&&' and '||'.*/
+var foo = a && b ? c : d;      /*BAD: Unexpected mix of '&&' and '||'.*/
```

Comment:
This comment seems incorrect.

Reply:
@platinumazure Both comments have been addressed